tests/pytests/test_gc.py

```diff
@@ -423,12 +439,21 @@ def test_gc_oom(env:Env):
     # Add some documents
     for i in range(num_docs):
         env.expect('HSET', f'doc{i}', 't', f'name{i}').equal(1)
+
+    # Get open file count
+    redis_server_pid = env.executeCommand('info')['process_id']
+    open_files_before_gc = get_open_file_count(redis_server_pid)
+    env.assertGreater(open_files_before_gc, 0)
```

Comment:
Why do we need this assertion?

Reply:
How do you know that there will be open fd's?

Reply:
`lsof -p <pid>` counts all open file descriptors for the Redis process, including stdin, stdout, stderr, listening socket, and more.

Reply:
But you are right, the CI failed. I'll remove it.